pkg/doc/verifiable/cache_js_wasm.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifiable
+
+import "time"
+
+// NewExpirableSchemaCache creates new instance of ExpirableSchemaCache.
+func NewExpirableSchemaCache(size int, expiration time.Duration) *ExpirableSchemaCache {
```

Comment:
I guess this function represents the default Cache implementation.

Reply:
yes